app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/IndividualHealthRecordFragment.kt

```diff
@@ -257,19 +260,22 @@ class IndividualHealthRecordFragment : Fragment(R.layout.fragment_individual_hea
         }
     }
 
-    private fun updateUi(uiState: IndividualHealthRecordsUiState) {
+    private suspend fun updateUi(uiState: IndividualHealthRecordsUiState) {
```

Comment:
remove suspend modifier from this method

Reply:
removed suspend function in the view layer